hadoop-tools/hadoop-aliyun/src/main/java/org/apache/hadoop/fs/aliyun/oss/AliyunOSSFileSystemStore.java

```diff
@@ -147,6 +147,11 @@ public void initialize(URI uri, Configuration conf, String user,
       throw new IllegalArgumentException(msg);
     }
 
+    boolean redirectEnable = conf.getBoolean(REDIRECT_ENABLE_KEY,
+    REDIRECT_ENABLE_DEFAULT);
+    clientConf.setRedirectEnable(redirectEnable);
+    LOG.debug("oss redirectEnable " + redirectEnable);
```

Comment:
prefer using {} expansion sin the log
```
LOG.debug("oss redirectEnable {}", redirectEnable);
```

1. saves building the string when debug is not enabled
2. the toString() call on every argument is very robust, handles raised exceptions and more. Not an issue for boolean, but stull a good habit